README.md

```diff
@@ -15,7 +15,7 @@ let config = JetstreamConfig {
 };
 
 let jetstream = JetstreamConnector::new(config).unwrap();
-let receiver = jetstream.connect().await?;
+let receiver = jetstream.connect(RetriesConfig::new()).await?;
```

Comment:
just curious if there's a use-case to customize the retry config on each `.connect()` call -- if not, would it make sense to put the retry config into JetstreamConfig? just thinking that could  be a less-breaking change.

Reply:
moved the configuration to  JetstreamConfig and added the configuratio paramter defining how long a connection must remain alive for the retries counter to be reset.